groups/bsl/bslstl/bslstl_optional.cpp

```diff
@@ -0,0 +1,29 @@
+// bslstl_optional.cpp                                                -*-C++-*-
+
+// ----------------------------------------------------------------------------
+//                                   NOTICE
+//
+// This component is not up to date with current BDE coding standards, and
+// should not be used as an example for new development.
+// ----------------------------------------------------------------------------
```

Comment:
It's from NullableValue, but I left it because I'm not sure that this component is/will/can be up to date with the current BDE coding standards - it does have to follow the standard library naming convention and interface. Shall I remove it ?

Reply:
removed from .h, .cpp and  .t.cpp